### ISSUE TITLE
How to create a floating window

### ISSUE THREAD

**Issue by elitetyc** [wailsapp/wails#3664]
### Have you read the Documentation Contribution Guidelines?

- [X] I have read the [Documentation Contribution Guidelines](https://wails.io/community-guide#ways-of-contributing).

### Description

How to create a floating window, on top of macos, if I open the browser in full screen, and then my wailsapp listens for shortcuts, and then calls runtime.Show, which will jump to the desktop to show my wails app, how can I do it so that I can show this app directly on the current full screen page?

### Self-service

- [X] I'd be willing to address this documentation request myself.

**Comment by OlegGulevskyy** [comment]
I don't think it exists yet, this needs to be implemented. In fact I am in the middle of doing it because my app requires such functionality (covering full screen without being in a full screen mode, displaying it on top of everything else).
I have a working POC thanks to @tmclane , but need to test it more before submitting a PR to Wails

**Comment by bruxaodev** [comment]
In the way ls3 has the `Frameless` and `AlwaysOnTop` properties, wouldn't that solve it?

**Comment by OlegGulevskyy** [comment]
> In the wails3 has the `Frameless` and `AlwaysOnTop` properties, wouldn't that solve it?

No, you can't make window go on top of the menu bar with these properties. You need to set [window level](https://developer.apple.com/documentation/appkit/nspopupmenuwindowlevel)
